python/lsst/ip/isr/overscan.py

```diff
@@ -523,6 +534,13 @@ def debugView(self, image, model):
         axes.plot(indices, plotModel, 'r-')
         plot.xlabel("centered/scaled position along overscan region")
         plot.ylabel("pixel value/fit value")
+        if amp:
```

Comment:
So, it's the {{amp}} only used to get information for the string in the title? (just confirming)

Reply:
Yes, this is code I pulled from DM-26281 before marking that invalid.  Debugging overscan quickly becomes impossible without information about which amp you're looking at, which wasn't available before.  Passing the optional `amp` gives information to make the title informative.